gradle.properties

```diff
@@ -9,4 +9,7 @@ android.useAndroidX=true
 
 #MPP
 kotlin.mpp.androidSourceSetLayoutVersion=2
-kotlin.mpp.enableCInteropCommonization=true
\ No newline at end of file
+kotlin.mpp.enableCInteropCommonization=true
+
+#Flavor
+flavor=dw
```

Comment:
This should probably go into the `local.properties` file that does not go into `git`.

Reply:
I did not want to load properties from `local.properties` since the file will be expected, so i removed the property and rely on the default or passing a gradle argument.